hypertrace-alert-engine/src/main/resources/configs/common/application.conf

```diff
@@ -15,14 +15,19 @@ job.config = {
 ruleSource = {
   type = "fs"
   fs = {
-    path = "build/resources/main/alert-rules.json"
+    path = "build/resources/test/configs/rules.json"
```

Comment:
Things will break over here, since both notification and task-manager service run as part of this

Reply:
I suggest we use a different block name `alertRuleSource` & `notificationRuleSource` for the services respectively

Reply:
same update name of the file